test/fixtures/v5/roundabout/exit_exit.json

```diff
@@ -0,0 +1,23 @@
+{
+    "step": {
+        "maneuver": {
+            "modifier": "left",
+            "type": "roundabout",
+            "exit": 1
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "In den Kreisverkehr fahren und erste Ausfahrt nehmen auf Way Name",
```

Comment:
German is kind of verbose on these. Wouldn't it be obvious to `enter the roundabout`? We could easily say `Im Kreisverkehr die erste Ausfahrt auf Way Name nehmen`, replacing `In den fahren` with `Im`.